lib/game.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Game do
+  @doc ~S"""
+  Prints a 10x10 board with row and column headers
+  """
```

Comment:
Is this doc string auto generated? I ask because if your board becomes dynamic and is the doc string still accurate?

Reply:
I'm not sure if doc strings are a convention in Elixir but the one thing to keep in mind is if/when something changes then the documentation will also need to be changed.

Reply:
The doc string was not auto generated, so I will have to change it when I change the method.

It looks to me like doc strings are the convention in Elixir.. You can apparently include "doc tests" inside them that will run as a part of the test suite, and they serve as examples of usage. I didn't do that for this method because I'm printing to the IO, but it would probably help catch outdated docs in other cases!